### PR TITLE
New package: budgie-desktop-10.5.1

### LINKED ISSUE #11815: Package Request: Budgie Desktop
Github Website:
https://github.com/solus-project/budgie-desktop

### PR DESCRIPTION
**Pull request by not-chicken**
closes #11815

### CODE REVIEW

**Comment by xtraeme** [comment]
did you test it? please provide some details (screenshots)

**Comment by not-chicken** [comment]
Tested on x86_64
budgie
budgie2

**Comment by xtraeme** [comment]
cool, thx for the info. Will look at the build logs

File: srcpkgs/budgie-desktop/template [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Template file for 'budgie-desktop'
+pkgname=budgie-desktop
+version=10.5.1
+revision=1
+archs="i686 x86_64*"
```
Comment (xtraeme) [inline srcpkgs/budgie-desktop/template]: please change this to `i686*`

Comment (xtraeme) [reply srcpkgs/budgie-desktop/template]: looks ok otherwise.

Comment (not-chicken) [reply srcpkgs/budgie-desktop/template]: Looks like it got merged before I pushed the changes.

**Comment by xtraeme** [comment]
yes, don't worry. we can change this later on.

Thanks!

File: srcpkgs/budgie-desktop/template (hunk 1 above)
Comment (Johnnynator) [reply srcpkgs/budgie-desktop/template]: Is this actually a issue with only supporting specific architectures or just cross compiling issues?

Comment (Duncaen) [reply srcpkgs/budgie-desktop/template]: It cross compiled fine for aarch64 and armv7l...

Comment (xtraeme) [reply srcpkgs/budgie-desktop/template]: we can enable cross compilation right now.

**Comment by Duncaen** [comment]
The package also contains development files (headers, pkg-config files)...

File: srcpkgs/budgie-desktop/template (hunk 1 above)
Comment (not-chicken) [reply srcpkgs/budgie-desktop/template]: > Is this actually a issue with only supporting specific architectures or just cross compiling issues?

I thought it was the former, which it seems isn't true. It cross compiles fine locally as Duncaen said.
So I should remove `archs` in next revision?

**Comment by not-chicken** [comment]
Sorry for missing that earlier and delay in responding to reviews. I'll fix `archs` and make a `-devel` subpackage asap in next revision. Thanks!

**Pull request merged**